build_modules/lib/src/meta_module.dart

```diff
@@ -89,10 +89,31 @@ class _AssetNode {
               'same package');
         }
         parts.add(linkedId);
-      } else if (internalSrcs.contains(linkedId)) {
-        internalDeps.add(linkedId);
-      } else {
-        externalDeps.add(linkedId);
+        continue;
+      }
+
+      List<Configuration> conditionalImportConfigurations;
```

Comment:
nit: `conditionalDirectiveConfigurations`? technically this handles imports and exports.

Reply:
Got it, renamed